tests/common/auth_test.go

```diff
@@ -118,7 +118,7 @@ func TestAuthGracefulDisable(t *testing.T) {
 
 		watchCh := rootAuthClient.Watch(wCtx, "key", config.WatchOptions{Revision: 1})
 		wantedLen := 1
-		watchTimeout := 10 * time.Second
+		watchTimeout := 15 * time.Second
```

Comment:
why is this test affected?

Reply:
Because I change `gs.Stop` to `gs.GracefulStop` in serving code's cleanup.

https://github.com/etcd-io/etcd/blob/0cd5999b7841856e269d66d2db877044da6bb240/server/embed/serve.go#L159-L163

ETCD calls `http.Shutdown` during `stopServers`.

https://github.com/etcd-io/etcd/blob/0cd5999b7841856e269d66d2db877044da6bb240/server/embed/etcd.go#L458-L465

That call will close the `net.Listener`. So both `http.Serve` and `grpc.Serve` will exit because of `using closed connection`. Before this patch, we always call `gs.Stop` in the following code. It's kind of conflict with `stopServers` logic which wants graceful shutdown. So, I change it and it takes a little longger than before. Hope it can help

https://github.com/etcd-io/etcd/blob/0cd5999b7841856e269d66d2db877044da6bb240/server/embed/serve.go#L159-L163